frontend/src/metabase/visualizations/visualizations/BarChart.jsx

```diff
@@ -18,6 +22,9 @@ export default class BarChart extends LineAreaBarChart {
   static iconName = "bar";
   static noun = t`bar chart`;
 
+  static minSize = getMinSize("area");
```

Comment:
I believe this is a typo. That applies to the `defaultSize` as well

```suggestion
  static minSize = getMinSize("bar");
```